packages/backend/src/open_payments/payment/outgoing/service.ts

```diff
@@ -146,9 +154,14 @@ async function getOutgoingPayment(
 ): Promise<OutgoingPayment | undefined> {
   const outgoingPayment = await OutgoingPayment.query(deps.knex)
     .get(options)
-    .withGraphFetched('[quote.asset, walletAddress]')
```

Comment:
We can also replace the `withGraphFetched` in the cancelOutgoingPayment function (mostly for consistency)

Reply:
Do we also want to do this for incomingPayments?